### PR TITLE
whatsnew for 3/30/20

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) updates the whatsnew YAML file with latest.

## Affected DevDocs pages

-  https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,7 +4,42 @@ description: |
...
+entries:
+- description: Added a section for the Tooltip Element to the [Magento UI library](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/theme-ui-lib.html)
+    topic.
+  versions: 2.3.x
+  type: Major Update
+  date: March 30, 2020
+  link: https://github.com/magento/devdocs/pull/6947
+- description: "Added a 'ignoreTmpls property' section to [Template Literals
+    in UI Components](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.html)
+    page."
```
Comment (meker12) [inline src/_data/whats-new.yml]:
```suggestion
- description: "Added an 'ignoreTmpls property' section to the [Template Literals
    in UI Components](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.html)
    topic."
```

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -18,7 +53,7 @@ entries:
   type: New topic
   date: March 23, 2020
   link: https://github.com/magento/devdocs/pull/6876
-- description: Added a new topic for the [`Container ui-component`](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-form.html).
+- description: Added a new topic for the [`Container ui-component`](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-container.html).
```
Comment (meker12) [inline src/_data/whats-new.yml]:
```suggestion
- description: Added a topic for the [`Container ui-component`](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-container.html).
```

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -34,7 +69,7 @@ entries:
   type: Major update
   date: March 16, 2020
   link: https://github.com/magento/devdocs/pull/6863
-- description: Added topic for [ColumnsResize UI](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-columns-resize.html).
+- description: Added topic for [ColumnsResize UI Compnent](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-columns-resize.html).
```
Comment (meker12) [inline src/_data/whats-new.yml]:
```suggestion
- description: Added topic for the [ColumnsResize UI Compnent](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-columns-resize.html).
```

File: src/_data/whats-new.yml [hunk 4]
```diff
@@ -4,7 +4,42 @@ description: |
...
+  versions: 2.3.x
+  type: Major Update
+  date: March 24, 2020
+  link: https://github.com/magento/devdocs/pull/6913
+- description: Added a 'Declaring the new custom no-route processor' section to the
+    [Routing](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/routing.html).
```
Comment (meker12) [inline src/_data/whats-new.yml]:
```suggestion
    [Routing](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/routing.html) topic.
```

File: src/_data/whats-new.yml [hunk 5]
```diff
@@ -4,7 +4,42 @@ description: |
...
+- description: Available now! Amazon Sales Channel 4.0 introduces a new [Amazon Sales
+    Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-sales-channel-home.html)
+    home page, a new [store dashboard](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-store-dashboard.html),
+    a streamlined [onboarding process](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-onboarding-home.html),
+    and new [default store settings](https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/default-store-settings.html).
+  versions: 2.3.x
+  type: Major Update
+  date: March 24, 2020
+  link: https://github.com/magento/devdocs/pull/6915
+- description: Magento has released a hot fix on March 24, 2020 for Magento 2.3.x.
```
Comment (meker12) [inline src/_data/whats-new.yml]: I updated this PR to have the same links that are in the release -- showing the text instead of the raw URL.

**Comment by dobooth** [comment]
running tests

**Pull request merged**